src/main/webapp/js/richTextEditor.js

```diff
@@ -37,11 +37,10 @@ var richTextEditorBuilder = {
         };
     },
 
-    initEditor: function(selector, inline) {
+    initEditor: function(selector, opts) {
         tinymce.init(jQuery.extend(this.getDefaultConfiguration(), {
```

Comment:
I got used to use $ only for selectors and variables. If it contradicts current policies (haven't seen that), I'll change it